cmd/config/internal/commands/source.go

```diff
@@ -50,7 +53,10 @@ func (r *SourceRunner) runE(c *cobra.Command, args []string) error {
 	// if there is a function-config specified, emit it
 	var functionConfig *yaml.RNode
 	if r.FunctionConfig != "" {
-		configs, err := kio.LocalPackageReader{PackagePath: r.FunctionConfig}.Read()
+		configs, err := kio.LocalPackageReader{
```

Comment:
How would sink react to this? Does it write json files out or yaml resources to .json files? Taking json files and writing out yaml files might be confusing to user.

Reply:
Copy the PR, then write a test and find out.